CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.0.1] - 2025-04-25
+### Changed
+- Downgrade to kallisto version 0.48.0.
```

Comment:
simply noting here that this was done to match the version currently being run at CHUM, due to our identifying differences in the output associated with using this more recent version of Kallisto.